tests/functional/test_collection_meta.py

```diff
@@ -292,6 +325,8 @@
             "collections -> bad.foo8 -> removal: Value error, new_name must not be provided if reason is not 'renamed'",
             "collections -> bad.foo9 -> removal: Value error, redirect_replacement_major_version must not be provided if reason is not 'renamed'",
             "extra_stuff: Extra inputs are not permitted",
+            "removed_collections -> bad.foo1 -> removal -> major_version: Input should be a valid integer, unable to parse string as an integer",
```

Comment:
We _could_ add a pre validator that specifically checks for `TBD` and gives a more helpful message in that case, but that might be overengineering.

Reply:
Should be no longer necessary due to 47593714d43b0fc8f73d6bbe00ab4f2acb511ae3.